src/core/sys/linux/sys/procfs.d

```diff
@@ -0,0 +1,15 @@
+/**
+ * D header file for GNU/Linux.
+ *
+ * License:   $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+ * Authors:   Teodor Dutu
+ */
+
+module core.sys.linux.sys.procfs;
+
+import core.sys.posix.sys.types : pid_t;
```

Comment:
What is the purpose of `version(linux)` if the file does not compile because of this import? There's no pid_t on windows, so the import will fail.
Move this line to the `version(linux)`
@teodutu